src/index.js

```diff
@@ -18,7 +29,9 @@ function render (TestComponent, { props = null, store = null, routes = null } =
   if (routes) {
     const VueRouter = require('vue-router')
     localVue.use(VueRouter)
-    router = new VueRouter(routes)
+    router = new VueRouter({
```

Comment:
This is the change. Had to pass routes as an object.

As outlined in this [example](https://lmiller1990.github.io/vue-testing-handbook/vue-router.html#writing-the-test)